app/views/ViewUtils.scala

```diff
@@ -26,6 +28,39 @@ object ViewUtils {
   }
 
   def breadcrumbTitle(title: String)(implicit messages: Messages): String = {
-    s"$title - ${messages("site.service_section")} - ${messages("site.service_name")} - GOV.UK"
+    s"$title - ${messages("site.service_section")} - ${messages("service.name")} - GOV.UK"
   }
+
+  def errorHref(error: FormError, radioOptions: Seq[RadioOption] = Nil): String = {
+    error.args match {
+      case x if x.contains("day") || x.contains("month") || x.contains("year") =>
+        s"${error.key}.${error.args.head}"
+      case _ if error.message.toLowerCase.contains("yesno") =>
```

Comment:
This has just been changed by @carrielies for maintain-a-protector-frontend

Reply:
Updated